laterpay/asset_sources/js/laterpay-account-links.js

```diff
@@ -0,0 +1,36 @@
+// render LaterPay login / logout links using the LaterPay YUI dialog manager library
+YUI().use('node', 'laterpay-dialog', 'laterpay-iframe', 'laterpay-easyxdm', function(Y) {
+
+    // check, if all required variables are available
+    if (!lpAccountLinksUrl || !lpAccountNextUrl) {
+        return;
+    }
+
+    // define URLs to forward the user to after login / logout / registration
+    var loginLink  = 'https://web.laterpay.net/auth/user/login?_on_complete=' + lpAccountNextUrl,
```

Comment:
where did you get this from? _on_complete should never be set outside of django-flows. The correct endpoints are
```
/account/dialog/login
/account/dialog/signup
/account/dialog/logout
```

as in [the LaterPay Python client](https://github.com/laterpay/laterpay-client-python/blob/develop/laterpay/__init__.py#L229). I'm actually alarmed that this works.